aws/cloudformation/standalone/marketing/deployment/oidc.yml.erb

```diff
@@ -0,0 +1,21 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Set up OIDC identity provider and IAM role for GitHub Actions
+
+Parameters:
+  MarketingApplicationStackName:
+    Type: String
+    Description: The name of the Marketing application stack to use for this deployment.
+
+Resources:
+  GitHubOIDCProvider:
```

Comment:
Should we move this to the [in-progress updates to the IAM Stack](https://github.com/code-dot-org/code-dot-org/pull/65584/files)? I can try doing that.

Reply:
That sounds good to me.